modules/azure/virtual-machine/main.tf

```diff
@@ -0,0 +1,56 @@
+locals {
+  dd_tags = {
+    Datadog                 = "true"
+    DatadogAgentlessScanner = "true"
+  }
+}
+
+resource "azurerm_orchestrated_virtual_machine_scale_set" "vmss" {
```

Comment:
Is there any ability to automatically restart VMs on a fixed period ? We do that for ASGs, instances are recycled every 24 hours.

It helps with auto-updating.

Reply:
I will leave this for later.

Reply:
I found [Start/Stop VMs v2](https://learn.microsoft.com/en-us/azure/azure-functions/start-stop-vms/overview) but it seems a bit overkill.